spec/spec_helper.rb

```diff
@@ -22,6 +22,7 @@ def self.remove_constants
 RSpec.configure do |config|
   config.disable_monkey_patching!
   config.filter_run_when_matching :focus
+  config.warnings = true
```

Comment:
@flash-gordon we can have it enabled via `-w` flag in the shared `.rspec` in devtools

Reply:
@solnic done